ztp/source-crs/extra-manifest/99-crio-disable-wipe-master.yaml

```diff
@@ -0,0 +1,17 @@
+# use `sudo crio-status config` to check the current configs
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-crio-disable-wipe-master
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:text/plain;charset=utf-8;base64,W2NyaW9dCmNsZWFuX3NodXRkb3duX2ZpbGUgPSAiIgo=
```

Comment:
I think you should instead be creating this in ztp/extra-manifests-builder - that will allow you to store the file in plain text and have the extra-manifest files generated automatically.

Reply:
Ah had no idea it was thing! Will update